docker/candlepin-base/cp-test

```diff
@@ -49,6 +49,29 @@ validate_translation() {
     fi
 }
 
+gettext() {
+    if [ -f ./gradlew ]; then
+        ./gradlew gettext
+    fi
+
+    #Code to commit the updated files
+    CHANGED_FILE=`git diff --name-only HEAD`
+    ADD_COUNT=0
+
+    for file in $(echo $CHANGED_FILE | tr "\n" "\n")
+    do
+      echo "Performing: git add $file."
```

Comment:
There's no need to loop and add each individual file here a `git add common/po` would suffice, since gettext only changes the keys.pot file.